README.md

```diff
@@ -9,37 +9,38 @@
   - https://github.com/reserveMe/reservation-calendar-service
   - https://github.com/reserveMe/reviews-proxy
 
-## Table of Contents
-
-1. [Usage](#Usage)
-1. [Requirements](#requirements)
-1. [Development](#development)
-
 ## Usage
 
-> 
 * Run `npm install` within the root directory.
 * Ensure Mongoose and Mongo are running. 
 * In order to seed the database with 3,015 entries of review data, run `npm run seed`. 
 * To start webpack, run `npm run react-dev`. 
 * To start the Express server on port 3004, run `npm run server-dev`. 
 * To run the test suite using Jest, run `npm run test`.
 
-## Requirements
-
-An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
-
-- Node 6.13.0
-- etc
 
-## Development
+## CRUD OPERATIONS
 
-### Installing Dependencies
+### CREATE
+* HTTP request method: POST
+* Endpoint: `/api/restaurants/:id/reviews/:qty`
+* Required Parameters: id and qty
+* Expected Output: An array of the reviews that were added to the db
 
-From within the root directory:
+### READ
+* HTTP request method: GET
+* Endpoint: `/api/restaurants/:id/reviews`
+* Required Parameters: id
+* Expected Output: An array of all the reviews for the specified restaurant
 
-```sh
-npm install -g webpack
-npm install
-```
+### UPDATE
+* HTTP request method: PUT
```

Comment:
i think the actual use case is adding a single review (by date) -- so the entry needs to be inserted in the right place sequentially.

Reply:
You're right, I wasn't thinking of a proper use case. I was thinking of how I could conveniently create a PUT route to update the reviews based on the fake data. I'll build out a better UPDATE API for the new databases.